API/contact/ContactAPI.php

```diff
@@ -0,0 +1,148 @@
+<?php
+    require_once __DIR__ . '/../utils/JsonUtils.php';
+
+    class ContactAPI 
+    {
+        private mysqli $mysql;
+
+        public function __construct(mysqli $mysql)
+        {
+            $this->mysql = $mysql;
+        }
+
+        public function __destruct()
+        {
+            $this->mysql->close();
+        }
+
+        /**
+         * Creates contact record.
+         * 
+         * @param object $contact object of the Contact class.
+         * @return object|false object of the Contact class containing all information about created record or false if operation was unsuccessful. 
+         */
+        public function CreateContact(object $contact) : object|false 
+        {
+            if ($this->mysql->connect_error != null)
+                return false;
+
+            $stmt = $this->mysql->prepare("INSERT INTO Contacts (ID, firstName, lastName, email, phone, userID) VALUES(DEFAULT, ?, ?, ?, ?, ?)");
+            $stmt->bind_param(
+                "sssss", 
```

Comment:
I am not super knowledgeable about php/sql. Is it correct for all of these to be strings? I thought userID is an int but don't remember.

Reply:
well, it doesn't throw an error, but you are right, for the safety I will change it to int in the next commit